lib/braveAlerter.js

```diff
@@ -37,7 +45,6 @@ class BraveAlerter {
   }
 
   /* eslint-disable class-methods-use-this */
-  // TODO Make this a static function and remove the ESLint exception, but don't break its callers
```

Comment:
Removed this TODO because eventually this function will become the one that will add Notifications for the Alert App, at which point it will need a DB function, which will be a `this.something` function here. So we shouldn't do the work to make it static because we'll just need to undo it.

Reply:
@schwarrrtz and I talked about this and it's now clear that the function to make notifications and the one that is used for changing an alert type (from BUTTONS_NOT_URGENT to BUTTONS_URGENT, say) should be different functions since they will have different behaviour in the Alert App.